nixos/doc/manual/release-notes/rl-2305.section.md

```diff
@@ -68,6 +68,8 @@ In addition to numerous new and upgraded packages, this release has the followin
 
 - [imaginary](https://github.com/h2non/imaginary), a microservice for high-level image processing that Nextcloud can use to generate previews. Available as [services.imaginary](#opt-services.imaginary.enable).
 
+- [oelint-adv](https://github.com/priv-kweihmann/oelint-adv), a advanced linter for BitBake recipes based on oelint.
```

Comment:
This part of the release notes is only for new services, not new packages. As far as I know we don't add new packages to the release notes

Reply:
Well, there are some other tools/packages listed, like [trurl](https://github.com/NixOS/nixpkgs/pull/224526), which was merged not that long ago.

But I'll remove it's preferred, I fine with either way.